cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -251,6 +262,8 @@ public String commitAndPush(final CommitMeta commitMeta,
       }
     }
 
+    metricsContext.gauge(SourceControlManagement.METRIC_COMMIT_PUSH_LATENCY,
```

Comment:
a gauge measures some value as it changes over time, like a temperature. It doesn't seem to be what we want here, as previous values will be overwritten by newer values.

It seems more useful to have a counter for total push latency as well as a counter for total push operations, which can give us an average.

Reply:
Yes, using two counters seems like a better approach. We want rolling averages of the value with time. Will make the changes.

Reply:
Switched to using counter metrics.